connect/README.md

```diff
@@ -0,0 +1,259 @@
+<h1 align="center">⚡️ hyper connect workshop ⚡️</h1>
+<p align="center">In this workshop, we will dive deeper into hyper-connect and learn some more Data service methods</p>
+
+---
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [What is hyper-connect?](#what-is-hyper-connect?)
+- [Setup](#setup)
+- [Update a character](#update-a-character)
+- [Remove a character](#remove-a-character)
+- [List characters](#list-characters)
+- [Summary](#summary)
+
+---
+
+## Introduction 
+
+👋🏻 Greetings, Thank you for taking the time to learn hyper with us, in this workshop, we
+will dive a little deeper into hyper-connect the deno/nodejs hyper client. But first, if you
+have not taken the [Hello World](../hello-world) workshop, please consider to take that workshop
+first, we will be building on the concepts we learned in previous workshops.
+
+> NOTE: Please be sure to take the [Hello World](../hello-world) workshop before taking this workshop.
+
+As we started in the hello-world workshop, we will be building out a "Mario Wiki API", and through these
+workshops, we will cover all of the hyper core services. The best way to learn is by doing, so in order 
+to get the most out of these workshops, take the time to actually do the work, not just watch or copy
+and paste.
+
+The hyper Data Service is a JSON API that focuses on Document Data Storage, document stores gives
+developers the ability to keep their business logic clean separated from their services. With our
+JSON API, you can create, read, update and delete JSON Documents. In the previous workshop, we covered
+create and read, in this workshop, we will cover update, delete and list. 
+
+---
+
+## Prerequisites
+
+What do I need to know to take this workshop?
+
+- HTTP/JSON APIs 
+- Javascript (async/await promises)
+- git/Github (you will need a github account)
+
+> NOTE: We will be using deno a Javascript server runtime for this workshop, https://deno.land
+
+---
+
+## What is hyper-connect?
+
+hyper-connect is a deno/nodejs client module that you import into your application as a 
+dependency to provide some syntatic sugar to connect with the hyper API. The client takes
+a connection string that is provided via an environment variable to build http client calls.
+
```

Comment:
`hyper-connect` is a deno/nodejs client module that you import into your application as a 
dependency to provide some syntactic sugar to connect with the hyper API. The client takes
a connection string that is provided via an environment variable to build HTTP client calls.

Reply:
syntactic was misspelled.  corrected above in the comment.